syntax = "proto3";

package protocol.mmo;
import "MMO_enum.proto";
import "MMO_struct.proto";

message EnterGame_Resp
{
	E_RespCode resp = 1;
	ObjectInfo myInfo = 2;
}
message ObjectEnter_Notify
{
	ObjectInfo otherInfo = 1;
}
message ObjectExit_Notify
{
	int32 otherId = 1;
}
message Move_Req
{
	E_Dir dir = 1;
}
message Move_Notify
{
	int32 id = 1;
	E_Dir dir = 2;
}